locales/en-US/notes.properties

```diff
@@ -1,4 +1,12 @@
-welcomeTitle3=Welcome to Firefox Notes!
+## “Notes” and “Firefox” must be treated as a brand, and kept in English.
+## They cannot be:
+## - Declined to adapt to grammatical case.
+## - Transliterated.
+## - Translated.
+-brand-notes = Firefox Notes
+-brand-notes-short = Notes
+
+welcomeTitle3=Welcome to { -brand-notes }!
```

Comment:
Please keep this one as it was. To change it, you would need a different ID (e.g. welcomeTitle4), and it would invalidate all existing translations.

The idea is to use the new approach for new strings.

Reply:
Makes sense, will revert that back.